src/verification_key.rs

```diff
@@ -48,7 +48,7 @@ impl TryFrom<&[u8]> for VerificationKey {
     ///
     /// * `Result<Self, Self::Error>` - A VerificationKey if deserialization succeeds, or a VerifyError if it fails.
     fn try_from(value: &[u8]) -> Result<Self, Self::Error> {
-        VerificationKey::deserialize_compressed(value)
+        VerificationKey::deserialize_compressed_unchecked(value)
```

Comment:
Would it be possible to create a separate function with the checked deserialization ?

Reply:
Done in https://github.com/HorizenLabs/proof-of-sql-verifier/pull/5/commits/37a5eef380c0f0eb904e1b5ee729396d9076176e